src/engine/Service.cpp

```diff
@@ -92,6 +95,44 @@ ResultTable Service::computeResult() {
   serviceIriString.remove_suffix(1);
   ad_utility::httpUtils::Url serviceUrl{serviceIriString};
 
+  if (siblingTree_ != nullptr) {
```

Comment:
This should be in its own function that returns e.g. an `optional<string>` that returns the VALUES clause or nothing.